### PR TITLE
OOP Library/ Unit tests

### PR DESCRIPTION
**Pull request by Munyabelden**
### PR Description

- [ ] Install rspec and add to the Gemfile
- [ ] Create a folder name specs to store all my test files
- [ ] Tested classes:
         - Person
         - Student
         - Classroom
         - Rantal
         - Book
         - Teacher
         - CapitalizeDecorator
         - TrimmerDecorator
 
**Thank you for the review😊**

### CODE REVIEW

File: specs/person_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+require_relative '../person'
```
Comment (Munsa1) [inline specs/person_spec.rb:1]: - [ ] It's a requirement to make sure to check there is an 'it' block for each class method (at least). Kindly add this for the `add_rental` method.

File: specs/capitalize_decorator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require_relative '../decorator'
+require_relative '../capitalize_decorator'
```
Comment (Munsa1) [inline specs/capitalize_decorator_spec.rb:2]: - [ ] Kindly add tests for all classes that you created while implementing the Decorator design pattern as this is a requirement.
Please add tests for the `nameable..rb` class

image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Munyabelden @Tracy-miranj, 
required-changes-1


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
All Passing Tests :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive Branch Name :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Munyabelden @Tracy-miranj, 
required-changes-1


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
All Passing Tests :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive Branch Name :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @Munyabelden, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**